app/views/help.scala.html

```diff
@@ -10,81 +10,231 @@ <h2 class="govuk-heading-m">This user help guide includes:</h2>
+                TDR is being built using an Agile approach.
+                Private beta is an early Agile stage where the service is still being developed and does not yet offer the full range of features that we intend to support.
+                For example, TDR can currently only accept records that are open (i.e. no FOI exemptions apply), crown copyright and in English.
             </p>
             <p class="govuk-body">
-                At this private beta stage, the service is also only available to an invited group of users. Over time, we will make TDR available to more public record bodies and will continue to develop the service in consultation with our users, to add support for closed records and more complex types of records.
+                At this private beta stage, the service is also only available to an invited group of users.
+                Over time, we will make TDR available to more public record bodies and will continue to develop the service in consultation with our users, to add support for closed records and more complex types of records.
             </p>
             
             <h2 class="govuk-heading-m" id="before-you-begin">Before you begin</h2>
             <p class="govuk-body">
-                This service is currently in private beta, an early agile stage of development where limited features are available. Please be aware that at this stage, we can only accept records that are Crown Copyright, in english, open (i.e no FOI exemptions apply) and from your shared drive. 
+                This service is currently in private beta, an early agile stage of development where limited features are available.
+                Please be aware that at this stage, we can only accept records that are Crown Copyright, in English, open (i.e. no FOI exemptions apply) and from your shared drive.
             </p>
             <p class="govuk-body">
                 Over time, we will extend the service to accept a wider range of records such as those in a different language, closed records and other complex types of records.
             </p>
+
+            <h3 class="govuk-heading-s" id="before-web-browsers">Web browsers</h3>
             <p class="govuk-body">
-                To ensure you have the best experience, and can use all of our available features, we recommend you use the latest version of Google Chrome. Please be aware that Internet Explorer is not currently supported at all. 
+                We do not support Internet Explorer (IE11).
+                We advise you to use a different browser such as Edge, Chrome, Safari or Firefox to ensure you can use this service and all of its features.
```

Comment:
Should we add Safari and a browser we advise them using? I remember it having a few weird issues.

Reply:
Do you mean remove Safari from the recommended list of browsers? If so I think this is a question for Kirsten/Linda/Pauline. Added it to the JIRA task

Reply:
Removed as per Linda's comment